controllers/ocsinitialization/sccs.go

```diff
@@ -66,6 +67,7 @@ func newRookCephCSISCC(namespace string) *secv1.SecurityContextConstraints {
 	}
 
 	scc, _ := cephcsi.NewSecurityContextConstraints(rookValues)
+	scc.RequiredDropCapabilities = []corev1.Capability{"ALL"}
```

Comment:
ideally, this change should have gone into cephcsi not here (but this is also acceptable as a short-term solution), Have we tested this change and ensured that no regression in PVC workflow? if yes, we can get this in.

Reply:
Agreed, this should be done in cephcsi so upstream gets the change as well. Can we do that now? For 4.17 we still have time. 

Anytime privileges are reduced, it can easily cause a regression. We will need confirmation that all relevant scenarios have been tested.

Reply:
@ShravaniVangur Can you pls test all required scenarios with this change then you can implement the change in the upstream APIS and bring them here in the ocs.

Reply:
Sure will make these changes in cephcsi. 

The following have been tested for with the changes in this PR: 
- Successful deployment of storagecluster. 
- Creation of PVCs for cephfs and rbd following which pods were created with the respective pvcs.
- Edited the PVC size.
- Taking a snapshot.
- Cloned the PVC.
At each point the provisioner/plugin logs were checked to ensure success of the action. 

@Madhu-1  are there any other PVC tests to be done in upstream or are the above-mentioned checks in downstream enough?

Reply:
Thanks for testing above, This should be good enough